README.md

```diff
@@ -124,12 +124,39 @@ request.applePayDisabled = !canMakePayments
 
 ### 3D-Secure + Drop-In
 
-The new Drop-In supports 3D-Secure verification. If you have enabled 3D-Secure in the control panel, then just enable it in the BTDropInRequest and set an amount.
+The new Drop-In supports 3D-Secure verification. If you have enabled 3D-Secure in the control panel, enable it in the BTDropInRequest and set an amount. Then, create a BTThreeDSecureRequest() object, setting as many fields on it as possible; the more fields that are set, the less likely it is that a user will be be presented with a challenge. Make sure to attach this object to the BTDropInRequest before use.
 
 ```swift
-let request =  BTDropInRequest()
+let request = BTDropInRequest()
 request.threeDSecureVerification = true
 request.amount = "1.00"
```

Comment:
So, I'm always confused about `request.amount` vs. `threeDSecureRequest.amount`, and I have a feeling merchants will be too. My understanding is that we want merchants to use `threeDSecureRequest.amount`, not `request.amount`. So, maybe we should just remove `request.amount` from this code snippet?

I know it's not part of this PR, but shouldn't we also mark `request.amount` as deprecated?

Reply:
^ That's what I did in the web version of this, so yeah, I think we should do that too to be consistent.

Reply:
I'll open a separate PR to deprecate `request.amount`.